src/dda/env/dev/types/linux_container.py

```diff
@@ -166,11 +161,8 @@ def start(self) -> None:
             if self.app.telemetry.api_key is not None:
                 env[AppEnvVars.TELEMETRY_API_KEY] = self.app.telemetry.api_key
 
-            if git_user := (self.app.config.git.user.name or get_git_author_name()):
-                env[GIT_AUTHOR_NAME_ENV_VAR] = git_user
-
-            if git_email := (self.app.config.git.user.email or get_git_author_email()):
-                env[GIT_AUTHOR_EMAIL_ENV_VAR] = git_email
+            env[GitEnvVars.AUTHOR_NAME] = self.app.config.user.name
+            env[GitEnvVars.AUTHOR_EMAIL] = self.app.config.user.email
```

Comment:
We need to keep the conditionals because we don't want to set these if they are empty strings.